crates/sui-core/src/authority.rs

```diff
@@ -2301,6 +2302,8 @@ impl AuthorityState {
                     "handle_consensus_transaction UserTransaction",
                 );
 
+                self.record_canonical_certificate_maybe(&certificate).await?;
```

Comment:
@LefKok here's where validators process consensus transactions from narwhal, they will get user sign from certificate. According to the logic in `record_canonical_certificate_maybe`, only the first tx from narwhal will be recorded user sig